3 - PB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1796,6 +1796,33 @@ Questi ultimi dovranno agire di conseguenza risolvendo i problemi emersi o piani
 
 Per avere traccia degli elementi verificati che costituiscono il prodotto, su GitHub è possibile visualizzare l'insieme delle pull request approvate ed integrate mediante merge.
 
+== Processo di Validazione <processo_validazione>
+
+_Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.4.11_
+
+=== Scopo
+
+Il processo di Validazione serve a fornire prove oggettive che il sistema, quando in uso, soddisfi i requisiti esposti dal Proponente descritti nell'#adr.
+
+Lo scopo del processo è quindi quello di accertare l'abilità del sistema, o di un suo specifico elemento, di soddisfare in determinate specifiche condizioni operative l'obbietivo per cui è stato creato.
+
+=== Risultati
+
+Come risultato della corretta implemenazione del processo di Validazione otteniamo:
+- individuazione di limiti e vincoli che influenzano i requisiti, il design e l'archiettura;
+- feedback da parte del Proponente in merito al lavoro svolto;
+- evidenza che il sistema o l'elemento di sistema realizza i bisogni del Proponente.
+
+=== Attività
+==== Preparazione alla validazione
+La validazione viene svolta prima internamente al gruppo, il Programmatore deve fornire, su indicazione del Progettista, un elemento software che viene quindi revisionato dal Verificatore come descritto nei paragrafi del processo di implementazione (@processo_implementazione), di integrazione (@processo_integrazione), e di verifica (@processo_verifica). Il sistema si ritiene quindi in uno stato accettabile, anche se incompleto, e pronto per essere mostrato al Proponente.
+
+==== Effettuare la validazione
+Il gruppo si ritrova durante dei meeting definiti settimanalmente con il Proponente per discutere dello stato di avanzamento dei lavori. Qui viene quindi mostrato il sistema in funzione (definito come MVC (_Minimum Viable Product_) in quanto il sistema potrebbe ancora non essere del tutto refinito e presentare problematiche in determinate circostanze o assenze di funzionalità perché ancora non del tutto implementate), con particolare enfasi sulle novità introdotte e le difficoltà incontrate. Il Proponente fornisce quindi feedback riguardo l'adempimento dei bisogni espressi, e aiuta il gruppo chiarendo particolari inerenti la realtà di riferimento a cui il sistema è indirizzato.
+
+==== Gestire i risultati della validazione
+Su indicazione del Proponente l'elemento software viene modificato qual'ora vengano riscontrate cricità influenzando eventualmente architettura, design e requisiti.
```

Comment:
in generale secondo me suona un po male, metterei: In seguito alle direttive del Proponente, l'elemento software viene adattato per affrontare eventuali criticità emerse durante il processo, potenzialmente influenzando l'architettura, il design e i requisiti.